hasher-matcher-actioner/hmalib/common/actioner_models.py

```diff
@@ -160,20 +186,20 @@ def from_aws_message(cls, message: str) -> "ReactionMessage":
             parsed["ContentKey"],
             parsed["ContentHash"],
             [BankedSignal.from_dict(d) for d in parsed["MatchingBankedSignals"]],
-            ThreatExchangeReactionLabel(parsed["ReactionLabelValue"]),
+            ReactionLabel(parsed["ReactionLabelValue"]),
```

Comment:
Be careful when merging... to_ and from_aws_message are no longer present in this class in master. And, by the way, it's moved to message_models.py. Note that from_match_message_and_label() _is_ still exists.